src/utils/Validate/validatePartialCollection.js

```diff
@@ -0,0 +1,25 @@
+// Local imports
+
+import validateComponent from "./validateComponent";
+import Utils from "..";
```

Comment:
This sets up a circular dependency. I think you ought to be doing this instead:
```suggestion
import PartialCollection from '../PartialCollection';
```

And then adjusting line 14 accordingly.